CHANGES

```diff
@@ -4,6 +4,11 @@
 
  Changes between 1.0.2g and 1.1.0  [xx XXX xxxx]
 
+  *) --strict-warnings no longer enables runtime debugging options
+     such as REF_DEBUG. Instead, REF_DEBUG is enabled automatically
```

Comment:
REF_DEBUG is the only one moved from --strict-warnings to --debug (BIO_DEBUG was just deleted). But the change is meant to stick for the future as well. Not sure how to word it better.